### DISCUSSION TITLE
FullStack .NET

### DISCUSSION THREAD

**Discussion by RafaelDifini** [balta-io/forum#618] [Tira Dúvidas] [answered]
Balta, estou seguindo o curso Fullstack .net, porem estou fazendo o projeto com entidades personalizadas, então antes de chegar na parte do blazor, ja criei um LoginRequest, pois criei meu endpoint de login como na imagem: 
![image](https://github.com/user-attachments/assets/e93e91b7-4020-4712-b8f0-dd0f8b6e6d4d), o estranho, é que quando eu descomento o mapeamento dos endpoints do identity aqui: 
![image](https://github.com/user-attachments/assets/1b335baa-5d82-4f14-a35d-438967ff27d8)
 o login pelo endpoint do identity, não funciona,  ele fica tentando receber um LoginRequest, oque não fez sentido pra mim, pois mesmo comentando toda essa parte de AccountHandler, a entidade LoginRequest, e o Mapeamento desse meu endpoint de LoginRequest, ele ainda fica esperando receber isso, print do Erro: 
![image](https://github.com/user-attachments/assets/24ada2a3-01b3-497b-9403-57f59eaab7f3)

Meu builder esta assim: 
![image](https://github.com/user-attachments/assets/7f676387-4ad1-4dc2-ac40-b1765cf8e769)

**Comment by henriqueweege** [comment]
Fala, @RafaelDifini ! td certo?
[Esse é repositório do projeto](https://github.com/RafaelDifini/FloripaSurfClubV2), né? O código está atualizado aqui no Git? Se não, tem como subir para eu dar uma olhada?
Valeeeu!

**Reply by RafaelDifini** [reply]
Opa, é sim! pronto, atualizado! @henriqueweege

**Reply by andrebaltieri** [reply, marked as answer]
Como está seu método de Login? E como está seu Post (JSON)?

**Reply by RafaelDifini** [reply]
Boa noite! meu metodo de login esta assim:

![image](https://github.com/user-attachments/assets/21338d91-1abf-4264-9e4e-00ef9623c684)

ele funciona!!! a questão é que comentando ele ou nao, o metodo de login do AddApiEndpoint padrão do indentity não funciona, memso comentando todas referencias possiveis para esse meu metodo @andrebaltieri

**Reply by RafaelDifini** [reply]
Boa noite, alguma ideia pessoal? @andrebaltieri , @henriqueweege

**Reply by andrebaltieri** [reply]
Tem algo errado nas rotas pelo que pude notar... ele está tentando sempre mandar o request para seu método customizado de login, correto?

Tenta debugar pra ver onde ele para e ter mais detalhes para nos mostrar...
